packages/components/src/components/hds/table/tr.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import type { HdsTableScope } from './types';
+import { HdsTableScope as HdsTableScopeValues } from './types.ts';
+import type { HdsFormCheckboxBaseSignature } from '../form/checkbox/base';
+
+export interface BaseHdsTableTrArgs {
+  Args: {
+    isSelectable?: boolean;
+    isSelected?: false;
+    selectionAriaLabelSuffix?: string;
+    selectionKey?: string | undefined;
```

Comment:
question: doesn't the `?` cover for `undefined`?
```suggestion
    selectionKey?: string;
```

Reply:
Yes it does. Good catch.